keyboards/dz60/keymaps/macos_arrow/keymap.c

```diff
@@ -12,60 +12,60 @@ enum my_keycodes {
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   /* ,-----------------------------------------------------------------------------------------.
-   * | Esc | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 0   | -   | =   | Backspace |
+   * |  `  | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 0   | -   | =   | Backspace |
    * |-----------------------------------------------------------------------------------------+
    * | Tab    | Q   | W   | E   | R   | T   | Y   | U   | I   | O   | P   | [   | ]   |   \    |
    * |-----------------------------------------------------------------------------------------+
-   * | _FL      | A   | S   | D   | F   | G   | H   | J   | K   | L   | ;   | '   | Enter      |
+   * | Esc/_FL  | A   | S   | D   | F   | G   | H   | J   | K   | L   | ;   | '   | Enter      |
    * |-----------------------------------------------------------------------------------------+
-   * | Shift     | Z   | X   | C   | V   | B   | N   | M   | ,   | .   | /   | _ML | Up  |Shift|
+   * | Shift     | Z   | X   | C   | V   | B   | N   | M   | ,   | .   | /   |     | Up  |Shift|
    * |-----------------------------------------------------------------------------------------+
-   * | Ctrl | Alt  | Gui  |                Space               |Gui/_ML| Alt | Left| Down|Right|
+   * | Ctrl | Alt  | Gui  |                Space               |  _ML  | Alt | Left| Down|Right|
    * `-----------------------------------------------------------------------------------------'
    */
   [_BL] = LAYOUT_2_shifts(
```

Comment:
I actually spent most of yesterday working on some changes for the DZ60 that will be in an upcoming PR. That PR has minor changes to your keymap.

Specifically, `LAYOUT_2_shifts` will be deprecated in favor of `LAYOUT_all`, because we're trying to have descriptive names for the keyboard layouts/matrices. Yesterday I was helping a user on the QMK Discord debug his DZ60 keymap, and in that process I discovered that the matrix given for `LAYOUT_all` on the DZ60 is currently wrong. When I fix it, it's identical to `LAYOUT_2_shifts`. 

All this is to say: This is fine now, but it'll probably be changed soon. Only thing that will be changed is that `LAYOUT_2_shifts` here will be changed to `LAYOUT_all`. Functionality will not change at all.

Reply:
Interesting. Can this bug explain an issue I have with my layout currently? On my `_ML` layout, the `9` of the numpad is shifted by one key on the right compared to were I placed it. I can't seem to address that key on this layer.

Reply:
Is this with the rev2 of your keymap, or this rev3? Either way, I can track that down and get you an answer.

Reply:
rev3, the one introducing numpad.

Reply:
The matrix used by `LAYOUT` (your `_FL` and `_ML` layers) doesn't match the one used by `LAYOUT_2_shifts`, so the firmware is actually looking for that key on a different switch than where you're intending it to be. That's why it's not on the correct key.

For that reason, I personally suggest that all layers in a keymap use the same matrix; so all the layers should be `LAYOUT_2_shifts` or `LAYOUT`, but not both.

Reply:
Oh right, good catch thank you. Let me push a fixed version.